### DISCUSSION TITLE
Incorrect redirection after 2FA

### DISCUSSION THREAD

**Discussion by 8qwe24657913** [community/community#67131] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I clicked an external link of [some issue page](https://github.com/intellij-rust/intellij-rust/issues/10655) where the 2FA kicked in.
After the 2FA, it redirected me to [https://github.com/_view_fragments/issues/show/intellij-rust/intellij-rust/10654/issue_layout](https://github.com/_view_fragments/issues/show/intellij-rust/intellij-rust/10654/issue_layout), which gave me a `403` (instead of `404`, indicating that the url may actually exist).
The latter url (especially the `_view_fragments` part) seems to be an internal one that shouldn't be exposed to users.
I suspected that to be a bug in the process of redirection after 2FA and therefore chose the `General` section. Should I post these in the `Projects and Issues` section as I'm accessing an issue page?

**Comment by patrykkawiak** [comment]
Try to clear your browser cache i think i should solve it

**Reply by patrykkawiak** [reply]
it could holds in memory old link structure or sth like that

**Reply by 8qwe24657913** [reply]
I can access the issue page via the original url. I'm just talking about the 2FA redirected me to another url that gives `403`s, so the redirection process seems to be buggy.

**Comment by Alluseri** [comment]
Just happened to me, now we have insight into GitHub's backend structure. lmao
